core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.spec.ts

```diff
@@ -149,10 +355,102 @@ describe('Opportunities List Component', () => {
       tick();
       mockReloadOpportunitiesEventEmitter.emit();
       tick();
+      component.gotoPage(1);
+      tick();
```

Comment:
Can you check the visible opportunities for the first page as well?

Reply:
done